utils/ansible/ftest/templates/daos-launch.sh.j2

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# shellcheck disable=SC1000-SC9999
+# shellcheck disable=all # Jinja2 template can not be parsed with shellcheck
```

Comment:
What is a Jinja2 template?

Currently our shellcheck code calls `file` on a file and looks for `shell script` which looks at the #! to identify files however https://github.com/daos-stack/daos/pull/13360 changes this and does it by file extension instead.

What I found was that the current checks actually miss a load of files so the above PR resolves that although for this file I think the current checks would find it but the new ones wouldn't, and therefore if we're adopting the new checks then this suppression could be removed - assuming this file genuinely isn't a shell script.

Reply:
Jinja2 is a file template generator used by ansible for generating configuration files, shell scripts, etc.
Here, it is used by ansible for generating bash scripts which will eventually be used for building DAOS and launching functional tests.
The jinja2 parts are confusing shellcheck, this is why I have disabled it.

Reply:
Another solution could be to remove the #! header.

Reply:
From my side, no real preference as both solutions have their pros and cons.
 @ashleypittman , tell me what you prefer or if you have another solution.

Reply:
I think the approach in this PR is correct for todays shellcheck checking, if my PR moves ahead then the suppressions here could (and possibly should) be removed.